reinforcement_learning/rl_cartpole_coach/rl_cartpole_coach_gymEnv.ipynb

```diff
@@ -576,7 +564,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.9"
+   "version": "3.6.5"
```

Comment:
are we downgrading python version?

Reply:
No, this is just a side-effect of saving the notebook on an instance with a lower python version. This will not change customer UX.